wdl/PBFlowcell.wdl

```diff
@@ -69,15 +69,21 @@ workflow PBFlowcell {
 
         File unaligned_bam = select_first([ExtractHifiReads.hifi_bam, unmapped_shard])
 
+        call PB.RemoveKinetics { input: bam = unaligned_bam }
```

Comment:
I know this might turn out to be a major headache, but should we do this removal optionally?
I know our PB data all has this, but we can imagine data sourced from other sequencers may not have it.